### PR TITLE
iOS ViewController test

### PR DESCRIPTION
**Pull request by badboy**
Based on #442 and #443

### CODE REVIEW

File: samples/ios/app/glean-sample-appUITests/MockServer.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
+//
```
Comment (Dexterp37) [inline samples/ios/app/glean-sample-appUITests/MockServer.swift]: Can we change this to the MPL license header?

File: samples/ios/app/pings.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This file defines the built-in pings that are recorded by glean telemetry. They are
```
Comment (Dexterp37) [inline samples/ios/app/pings.yaml]:
```suggestion
# This file defines the built-in pings that are recorded by the Glean SDK. They are
```

Comment (Dexterp37) [reply samples/ios/app/pings.yaml]: Should we change this in the rest of our YAML files?

File: samples/ios/app/glean-sample-app/AppDelegate.swift [hunk 3]
```diff
@@ -5,13 +5,20 @@
...
+
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     // swiftlint:disable line_length
     func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         let glean = Glean.shared
+
+        glean.registerPings(Pings.shared)
```
[the remark this answers is not in the document]
Comment (badboy) [reply samples/ios/app/glean-sample-app/AppDelegate.swift]: filing

[the remark this answers is not in the document]
Comment (badboy) [reply samples/ios/app/glean-sample-app/AppDelegate.swift]: https://bugzilla.mozilla.org/show_bug.cgi?id=1594326

**Comment by badboy** [comment]
I'm letting this sit, so @travis79 can take a look later today

**Comment by travis79** [comment]
> I'm letting this sit, so @travis79 can take a look later today

👀

**Pull request merged**